src/status_im/contexts/chat/messenger/messages/list/view.cljs

```diff
@@ -25,8 +24,8 @@
     [utils.re-frame :as rf]
     [utils.worklets.chat.messenger.messages :as worklets]))
 
-(defonce ^:const distance-from-last-message 4)
-(defonce ^:const loading-indicator-page-loading-height 100)
+(def ^:const distance-from-last-message 4)
+(def ^:const loading-indicator-page-loading-height 100)
```

Comment:
could you please elaborate?

Reply:
Sure! I explained this in the first commit: I did this change to make the code more REPL friendly, we can load this namespace into the REPL easier if it doesn't have a defonce.

Actually, a `defonce` for an integer doesn't make any difference compared to a `def`. `defonce` makes sense, for example, for an `atom`.

Reply:
thanks